### ISSUE TITLE
etcdserver: request timed out

### ISSUE THREAD

**Issue by rachid-casa** [etcd-io/etcd#11334]
We have an etcd cluster consisting of 5 nodes. (etcd-1, etcd-2, etcd-3,etcd-4 and etcd-5).
At 18:38, the etcd-4 was the leader. At this time an election was started by the etcd-5. A quorum of 3 nodes  (etcd-1, etcd-2 and etcde-5) was formed. The etcd-5 became leader. 10 seconds later, the etcd cluster was unhealthy. All etcd nodes reported "etcdserver: request timed out".
We want to understand why the etcd cluster  was unhealthy.
Does this have anything to do with the conflict on the index after the election of the etcd-5 as leader?
See sections from the logfile of the etcd nodes.

etcd Version: 3.3.11
Git SHA: 2cf9e51
Go Version: go1.10.3
Go OS/Arch: linux/amd64

OS:
CentOS Linux release 7.7.1908 (Core)

[etcd Incident.pdf](https://github.com/etcd-io/etcd/files/3815133/etcd.Incident.pdf)

**Comment by tytakoff** [comment]
@rachid-casa I'm going to try to reproduce this setup in AWS using a cluster of EC2 instances. Do I have to know any more details of your etcd cluster setup?

**Comment by rachid-casa** [comment]
No, i do not think so. In the attachement above in "etcd Incident.pdf" you will find the config parameters we use.
Thanks.
